scripts/global/tune_player.js

```diff
@@ -14,6 +14,7 @@
    */
   var createBooper = function(audioContext) { 
     
+	var instrumentName = "booper";
```

Comment:
Ugh, I messed up the indentation here... I'll have to fix this soon, but for now, I'm waiting for the next hourly bell.